bpf/bpf_lxc.c

```diff
@@ -1011,12 +1012,14 @@ __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_NAT46) int tail_ipv4_to_ipv6(struct
 	int ret;
 
 	if (!revalidate_data(skb, &data, &data_end, &ip4))
-		return DROP_INVALID;
+		return  send_drop_notify(skb, SECLABEL, 0, 0, 0, DROP_INVALID, TC_ACT_SHOT,
```

Comment:
Please introduce an inline function which calls revalidate_data() and IPv4_to_ipv6() and call send_drop_notify() in the error handler when calling that new function. Similar to how tail_handle_ipv6() does it. This way we do not inline the send_drop_notify() code twice in the prog section.

Reply:
Thanks for reviewing,